js/main.js

```diff
@@ -1,382 +0,0 @@
-  createElement("contactHeader", "h5", ["modal-title"], "contactTitle", "Contact the Developer");
-  createElement("contactHeader", "button", ["btn-close"], "contactHeaderBtn");
-  addModalCloseButton("contactHeaderBtn");
-  createElement("contactContent", "div", ["modal-body"], "contactBody", "Roger Mullins\nrogermullins.mba@gmail.com");
-  createElement("contactContent", "div", ["modal-footer"], "contactFooter");
-  createElement("contactFooter", "button", ["btn", "btn-secondary"], "contactFooterCloseButton", "Dismiss");
-  addModalCloseButton("contactFooterCloseButton");
-};
-
-function getZip() {
-  zipIn = document.getElementById("zip-in").value;
-
-  if (zipValidates(zipIn)) {
-    callAPI(zipIn);
-  } else {
-    alert("Invalid ZIP code.");
-  }
-
-  // Reset Input Box
-  zipIn.value = "";
-}
-
-function getTime() {
-  // Reused from alarm clock project -RM
-  let timeHours = 0;
-  let timeMinutes = 0;
-  let timeSeconds = 0;
-  let alarmHours = 0;
-  let alarmMinutes = 0;
-  let alarmSeconds = 0;
-  
-
-  let today = new Date();
-  let time = today.getHours() + ":";
-  if (today.getMinutes() < 10) {
-      time = time + "0";
-  }
-  time = time + today.getMinutes() + ":";
-  if (today.getSeconds() < 10) {
-      time = time + "0";
-  }
-  time = time + today.getSeconds();
-  let date = (today.getMonth()+1)+'-'+today.getDate() +'-'+ today.getFullYear();
-
-  let timeBox = document.getElementById("showTime");
-  let dateBox = document.getElementById("showDate");
-  timeBox.innerText = time;
-  dateBox.innerText = date;
-}
-
-function capitalizeConditions(stringIn) {
-  // Takes string of current conditions returned by API (all lower case)
-  // Returns same string with first letter capitalized
-  const firstLetter = stringIn.charAt(0);
-  const capitalized = firstLetter.toUpperCase();
-  const restOfString = stringIn.slice(1);
-  const stringOut = capitalized + restOfString;
-  return stringOut;
-}
-
-async function callAPI(zipIn) {
-  // Builds API call string and gets data from API
-  const apiKey = "a6063ef55e23ba5afdf671e31f5cdf4b";
-  const apiCall = ("https://api.openweathermap.org/data/2.5/weather?zip=" + zipIn + ",us&appid=" + apiKey);
-  
-  try {
-    const dataBack = await axios.get(apiCall);
-
-  // Update stateObj with returned data
-  stateObj.city = dataBack.data.name;
-  stateObj.zipCode = zipIn;
-  stateObj.tempK = (dataBack.data.main.temp+"K"); 
-  stateObj.tempF = (Math.floor((1.8*((dataBack.data.main.temp)-273))+32)+"°F");   // 1.8*(K-273) + 32 
-  stateObj.tempC = (Math.floor((dataBack.data.main.temp - 273.15))+"°C");         // Kelvin – 273.15
-  stateObj.condition = capitalizeConditions((dataBack.data.weather[0].main, " with ", dataBack.data.weather[0].description));
-  stateObj.imageURL = ("http://openweathermap.org/img/wn/"+dataBack.data.weather[0].icon+"@2x.png");
-  tempF = (Math.floor((1.8*((dataBack.data.main.temp)-273))+32)); // Used to change the temperature color background
```

Comment:
The outer `()` are unnecesary. 

Also the calculations could be better served in a function in case we needed to use them elsewhere.